pkg/cmd/experimental/router/router.go

```diff
@@ -93,22 +93,22 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 			case 1:
 				name = args[0]
 			default:
-				glog.Fatalf("You may pass zero or one arguments to provide a name for the router")
+				fmt.Errorf("You may pass zero or one arguments to provide a name for the router")
```

Comment:
`fmt.Errorf` does not print what you pass to it, it rather formats errors. Also I am not sure that you want to remove `Fatalf` since you probably want to exit after an error occurs. Is there any particular reason you want to remove `glog`? If so, you can use the `fmt` package (probably `Fprintf` and pass stderr to it) and `os.Exit` to accomplish the same thing `glog.Fatalf` is doing.